src/core/a-entity.js

```diff
@@ -145,12 +145,18 @@ var proto = Object.create(ANode.prototype, {
 
   setObject3D: {
     value: function (type, obj) {
+      var self = this;
       var oldObj = this.object3DMap[type];
       if (oldObj) { this.object3D.remove(oldObj); }
       if (obj instanceof THREE.Object3D) {
-        obj.el = this;
+        obj.el = self;
         this.object3D.add(obj);
       }
+      if (obj instanceof THREE.Group && obj.children.length) {
+        obj.traverse(function bindEl (child) {
```

Comment:
I think let's just not recurse for now, and do it if we find we have to. Don't want to unnecessarily do this.